prow/monitoring/alertmanager_servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: alertmanager
+    prow-app: monitoring
```

Comment:
`app`

Reply:
I think this one is OK.
Keep consistent with
https://github.com/kubernetes/test-infra/blob/87b7660d9786865bb6412fbddda0a555d29d4c73/prow/monitoring/prometheus_servicemonitor.yaml#L5-L6

Reply:
So they're both broken?

https://github.com/kubernetes/test-infra/blob/87b7660d9786865bb6412fbddda0a555d29d4c73/prow/monitoring/prow_prometheus.yaml#L31-L34

Reply:
We have both labels in the serviceMonitors: `app` and `prow-app`. So we should be good.
Yesterday, the target has been added into prometheus, right? ^_^

Reply:
My head hurts. Both seems fine, but also why bother with both?